sycl/include/CL/sycl/backend/level_zero.hpp

```diff
@@ -51,12 +51,20 @@ struct interop<backend::level_zero, accessor<DataT, Dimensions, AccessMode,
 
 namespace level_zero {
 
-// Implementation of various "make" functions resides in libsycl.so
+// Since Level-Zero is not doing any reference counting itself, we have to
+// be explicit about the ownership of the native handles used in the
+// interop functions below.
+//
+enum class ownership { transfer, keep };
```

Comment:
Suggest initializing enum values explicitly to avoid different automatic assigned values between libraries.
It should be OK to do it later.

Reply:
OK, will do in subsequent check-in to not re-do testing here.